Helper/Asset/AssetHelperRuntime.php

```diff
@@ -55,12 +55,26 @@ public function assets(string $hash, string $saveDir = 'bundles'): void
         }
     }
 
-    public function unzip(string $hash, string $saveDir): array
+    public function unzip(string $hash, string $saveDir, bool $mergeContent = false): array
     {
         try {
-            $this->extract($this->storageManager->getStream($hash), $saveDir);
+            $checkFilename = $saveDir . \DIRECTORY_SEPARATOR . sha1($saveDir);
```

Comment:
Should call the common storage manager to compute the hash (it might not be a sha1)

Reply:
Fixed